atomicapp/nulecule/base.py

```diff
@@ -200,6 +200,8 @@ def load_config(self, config=None, ask=False, skip_asking=False):
             config=config, ask=ask, skip_asking=skip_asking)
         if self.namespace == GLOBAL_CONF and self.config[GLOBAL_CONF].get('provider') is None:
             self.config[GLOBAL_CONF]['provider'] = DEFAULT_PROVIDER
+            logger.info("Provider not specified, using default provider - {}".
```

Comment:
ex ` logger.info("Provider not specified, using default provider - %s" % DEFAULT_PROVIDER`